### PR TITLE
Refine build.md

### LINKED ISSUE #1388: TestStdinParseOnly fails during local run
Following https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/build.md, I run the following command in the development Docker container.

```bash
SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
```

I go the following test failure.

```
=== RUN   TestStdinParseOnly
time="2019-12-10T19:49:46Z" level=info msg="sqlflowCmd: run locally" package=sql
time="2019-12-10T19:49:49Z" level=error msg="runSQLProgram error: failed: exit status 1\n==========Program==========\n\nfrom sqlflow_submitter.tensorflow.train import train\nimport tensorflow as tf\ntry:\n    import sqlflow_models\nexcept:\n    pass\n\nfeature_column_names = [\n\"sepal_length\",\n\n\"sepal_width\",\n\n\"petal_length\",\n\n\"petal_width\",\n]\n\nfeature_metas = dict()\n\nfeature_metas[\"sepal_length\"] = {\n    \"feature_name\": \"sepal_length\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nfeature_metas[\"sepal_width\"] = {\n    \"feature_name\": \"sepal_width\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nfeature_metas[\"petal_length\"] = {\n    \"feature_name\": \"petal_length\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nfeature_metas[\"petal_width\"] = {\n    \"feature_name\": \"petal_width\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\n\nlabel_meta = {\n    \"feature_name\": \"class\",\n    \"dtype\": \"int64\",\n    \"delimiter\": \"\",\n    \"shape\": [],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nmodel_params=dict()\n\nmodel_params[\"hidden_units\"]=[10,20]\n\nmodel_params[\"n_classes\"]=3\n\n\nfeature_columns = {\"feature_columns\": [tf.feature_column.numeric_column(\"sepal_length\", shape=[1]),\ntf.feature_column.numeric_column(\"sepal_width\", shape=[1]),\ntf.feature_column.numeric_column(\"petal_length\", shape=[1]),\ntf.feature_column.numeric_column(\"petal_width\", shape=[1])]}\n\ntrain_max_steps = 0\ntrain_max_steps = None if train_max_steps == 0 else train_max_steps\n\n\ntrain(is_keras_model=\"false\" == \"true\",\n    datasource=\"mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0\",\n    estimator=tf.estimator.DNNClassifier,\n    select=\"\"\"SELECT *\nFROM iris.train\n\"\"\",\n    validate_select=\"\"\"SELECT *\nFROM iris.train\n\"\"\",\n    feature_columns=feature_columns,\n    feature_column_names=feature_column_names,\n    feature_metas=feature_metas,\n    label_meta=label_meta,\n    model_params=model_params,\n    save=\"model_save\",\n    batch_size=1,\n    epochs=2,\n    verbose=0,\n    train_max_steps=train_max_steps,\n    eval_start_delay_secs=0,\n    eval_throttle_secs=0,\n    save_checkpoints_steps=100,\n    log_every_n_iter=10)\n\n==========Output==========\nTraceback (most recent call last):\n  File \"<stdin>\", line 100, in <module>\nTypeError: train() got an unexpected keyword argument 'train_max_steps'\n" package=sql
--- FAIL: TestStdinParseOnly (4.76s)
panic: interface conversion: interface {} is sql.EndOfExecution, not string [recovered]
	panic: interface conversion: interface {} is sql.EndOfExecution, not string

goroutine 51 [running]:
testing.tRunner.func1(0xc000193300)
	/usr/local/go/src/testing/testing.go:874 +0x3a3
panic(0x106f4a0, 0xc00030f410)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
sqlflow.org/sqlflow/cmd/repl.TestStdinParseOnly(0xc000193300)
	/sqlflow/cmd/repl/repl_test.go:282 +0xdde
testing.tRunner(0xc000193300, 0x124c4b8)
	/usr/local/go/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:960 +0x350
FAIL	sqlflow.org/sqlflow/cmd/repl	4.909s
```

### PR DESCRIPTION
**Pull request by shendiaomo**
Fix #1388

### CODE REVIEW

File: doc/build.md [hunk 1]
```diff
@@ -50,11 +50,13 @@ Then, we can build and run tests.
 
 ```bash
 go generate ./...
-SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
+PYTHONPATH=$GOPATH/src/sqlflow.org/sqlflow/python SQLFLOW_TEST_DB=mysql go test -v -q 1 ./...
```
Comment (wangkuiyi) [inline doc/build.md]: I got your point. Thanks. But I think it should be `PYTHONPATH=/sqlflow/python` in the Docker container. Am I correct?

Comment (wangkuiyi) [reply doc/build.md]: Also, it should be `-p 1` instead of `-q 1`.

Comment (shendiaomo) [reply doc/build.md]: > Also, it should be `-p 1` instead of `-q 1`.

3x && Updated

**Review by wangkuiyi — APPROVED** [review]

File: doc/build.md [hunk 2]
```diff
@@ -50,11 +50,13 @@ Then, we can build and run tests.
 
 ```bash
 go generate ./...
-SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
+PYTHONPATH=/sqlflow/python SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
```
Comment (typhoonzero) [inline doc/build.md:53]: Should be `PYTHONPATH=$GOPATH/src/sqlflow.org/sqlflow/python`, see `test_units.sh`

Comment (shendiaomo) [reply doc/build.md:53]: Good idea! I'll issue another pull request to improve.

**Pull request merged**